tests/php/Dev/DeprecationTest/DeprecationTestObject.php

```diff
@@ -14,7 +14,7 @@ public function __construct()
         Deprecation::withSuppressedNotice(function () {
             Deprecation::notice(
                 '1.2.3',
-                'Some class message in a future major release',
+                'Some class message',
```

Comment:
This should not have had "in a future major release" added to it and resulted in https://github.com/silverstripe/silverstripe-framework/actions/runs/14766287399/job/41458301799